test/jdk/sun/java2d/loops/GraphicsPrimitiveMgrTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2023, Azul Systems, Inc. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.util.concurrent.CountDownLatch;
+
+/**
+ * @test
+ * @bug 6995195
+ * @summary Verify that concurrent classloading of GraphicsPrimitiveMgr and Blit doesn't deadlock
+ * @run main/othervm/timeout=20 GraphicsPrimitiveMgrTest
+ */
+public class GraphicsPrimitiveMgrTest {
+
+    private static volatile CountDownLatch latch;
+
+    public static void main(final String[] args) throws ClassNotFoundException, InterruptedException {
```

Comment:
please split the long lines to 80 chars per line

Reply:
Fixed. Thanks.